bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/i18n/ConfigDescriptionGroupI18nUtil.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.config.core.i18n;
+
+import java.net.URI;
+import java.util.Locale;
+
+import org.eclipse.smarthome.core.i18n.I18nProvider;
+import org.eclipse.smarthome.core.i18n.I18nUtil;
+import org.osgi.framework.Bundle;
+
+/**
+ * The {@link ConfigDescriptionGroupI18nUtil} uses the {@link I18nProvider} to
+ * resolve the localized texts in the configuration parameter groups. It automatically infers the key if the default
+ * text is not a constant.
+ *
+ * @author Chris Jackson - Initial contribution
+ */
+public class ConfigDescriptionGroupI18nUtil {
```

Comment:
Do you have an example for the default keys?

I would expect something like this: thing-type.config.`<binding-id>.<thing-type-id>.<group-name>.label` and `<binding-id>.<thing-type-id>.<group-name>.description`